.github/workflows/ci.yaml

```diff
@@ -87,3 +87,24 @@ jobs:
     - run: .kokoro/build.sh
       env:
         JOB_TYPE: clirr
+  behavioral:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/checkout@v3
+      with:
+        repository: googleapis/cloud-bigtable-clients-test
+        ref: main
+        path: cloud-bigtable-clients-test
+    - uses: actions/setup-java@v3
+      with:
+        distribution: zulu
+        java-version: 11
```

Comment:
This should use 8 to match what we use in `maven-compile-plugin` https://github.com/googleapis/java-shared-config/blob/main/pom.xml#L169-L170